src/midgard/tiles.cc

```diff
@@ -70,6 +75,8 @@ template <class coord_t> struct closest_first_generator_t {
     queued.emplace(subdivision);
     queue.emplace(std::make_pair(0, subdivision));
     neighbors(subdivision);
+    constexpr int max_tested_corners = 8;
```

Comment:
Why is this 8? we're testing 4 corners, right? or maybe I'm missing something?

Reply:
never mind, I see we insert _at most_ 8.

Reply:
I dunno, I just counted the max number of things that were being added in the `dist()` function, and reserved that amount of space here.

Reply:
See here: https://github.com/valhalla/valhalla/pull/3324/files#diff-57cf239a14a55d61c34bbd9378f74eaebafeba721c2c466058337088192863d3L84